src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball (String color, int number) {

```

Comment:
According to the Java style guide, there should not be a space between the method name and the opening parenthesis.